about.css

```diff
@@ -366,19 +366,22 @@
 
   /* partner section start */
 
-  .partner_title {
-    font-size: 36px;
+  .mainProgramHeading-abt {
+    /* font-size: 36px; */
+    display: none;
   }
 
   .partner_logo {
-    display: grid;
+    display: none;
+
+    /* display: grid;
     grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
     padding: 0 100px;
     align-items: center;
     justify-content: center;
     padding-bottom: 30px;
     column-gap: 0;
```

Comment:
- [OPTIONAL] I think it would be better to remove unused code from your project rather than comment it out to keep your code clean and follow [the best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)💯

![image](https://user-images.githubusercontent.com/45231276/181750281-0d99a210-25c7-4b54-b455-141811a9c361.png)